recipe/build-mpi.sh

```diff
@@ -92,14 +101,18 @@ if [[ "$target_platform" == linux-* ]]; then
 fi
 
 ./configure --prefix=$PREFIX \
+            --enable-fast=O3 \
+            --enable-romio \
             --disable-dependency-tracking \
             --enable-cxx \
             --enable-fortran \
             --enable-f08 \
             --with-wrapper-dl-type=none \
             --disable-opencl \
-            --with-device=ch4 \
-            || cat config.log
+            --with-hwloc=$PREFIX \
+             --enable-nemesis-shm-collectives \
```

Comment:
Q: What does this do?

Reply:
I looked into it, and it's a feature for a ch3 device we don't use. Removed.